walk-in-interview/src/main/webapp/applicant-interested-list/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="../style.css">
+  </head>
+
+  <body>
+    <div class="header-container">
+      <button id="back"></button>
+    </div>
+
+    <div class="content-container">
+      <div id="interested-jobs-map"></div>
+
+      <div class="job-listings-container">
+        <div id="job-listings-title"></div>
+        <div id="job-listings-showing"></div>
+  
+        <ul id="job-listings"></ul>
+      </div>
+    </div>
+
+    <div class="error-message" id="error-message"></div>
+
+    <!-- hidden div to be used as a template to render job listings -->
+    <div hidden>
+        <li class="job-listing" id="job-listing-template">
+          <div class="job-name"></div>
+          <div class="job-address"></div>
+          <div class="job-pay"></div>
+          <div class="requirements-list"></div>
+          <form><input></form>
+        </li>
+    </div>
+  </body>
+
+  <script type="module" src="./script.js"></script>
+  <!-- TODO(issue/69): find a way to hide the api key -->
+  <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDXntH-1pvtC20SGfFhbzNPPagESHSR6E4&callback"
```

Comment:
since you are using the same api key everywhere, can you move it into a config.js file?